lib/jsonrpc2.ex

```diff
@@ -153,9 +153,10 @@ defmodule JSONRPC2 do
           | String.t()
           | [json]
           | %{optional(String.t()) => json}
+          | %{optional(atom()) => json}
 
   @typedoc "A JSON-RPC 2.0 params value."
-  @type params :: [json] | %{optional(String.t()) => json}
+  @type params :: [json] | %{optional(String.t()) => json} | %{optional(atom()) => json}
```

Comment:
Should this be `optional(String.t() | atom())` for the key type instead? Obviously hopefully not many mixed maps in the wild, but it is technically valid as well.

Reply:
Oh, I did not know that was also possible, thanks for the feedback, I have changed that.